### PR TITLE
Update staging with develop

### PR DESCRIPTION
**Pull request by ramabit**
# Description
Update staging with develop before creating release 0.0.2

### CODE REVIEW

File: src/services/badgeModels/badgeModels.ts [hunk 1]
```diff
@@ -33,15 +40,40 @@ export class BadgeModelsService extends TheBadgeSDKConfig implements BadgeModels
...
+   *
+   * @param badgeModelId
+   * @returns an Array<MetadataColumn> or null if not found
+   */
+  public async getEvidenceRequirementsOfBadgeModel(badgeModelId: string): Promise<Array<MetadataColumn> | null> {
+    // take ipfs uri from metadata of the badge model
+    const badgeModelMetadataResponse = await this.getMetadataOfBadgeModel(badgeModelId)
+    const ipfsDataUri = badgeModelMetadataResponse?.badgeModelKlerosMetaData?.registrationUri
+    if (!ipfsDataUri) {
+      return null
```
Comment (FedeMadoery) [inline src/services/badgeModels/badgeModels.ts:66]: I think that we should do `throw new Error("Missing RegistrationUri for the given model ID, provide a valid ModelID")`, return just `null` can be confusing. The same for the other cases where you are doing a check

File: src/services/badges/badges.ts [hunk 2]
```diff
@@ -162,4 +163,31 @@ export class BadgesService extends TheBadgeSDKConfig implements BadgesServiceMet
...
+   * @returns a string or null if not found
+   */
+  public async getImageIPFSHashOfBadge(badgeId: string): Promise<string | null> {
+    // get badge with the given id
+    const badgeResponse = await this.getById(badgeId)
+
+    // take ipfs uri from badge object
+    const ipfsDataUri = badgeResponse?.badge?.uri
+    if (!ipfsDataUri) {
+      return null
```
Comment (FedeMadoery) [inline src/services/badges/badges.ts:180]: Same as before

**Review by FedeMadoery — CHANGES_REQUESTED** [review]

File: src/services/badgeModels/badgeModels.ts (hunk 1 above)
Comment (ramabit) [reply src/services/badgeModels/badgeModels.ts]: fixed

File: src/services/badges/badges.ts (hunk 2 above)
Comment (ramabit) [reply src/services/badges/badges.ts]: fixed

**Review by FedeMadoery — APPROVED** [review]

**Pull request merged**